support/ci/rustfmt.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -euo pipefail
+
+export RUSTUP_HOME="/opt/rust"
+export CARGO_HOME="/home/buildkite-agent/.cargo"
+export PATH="/opt/rust/bin:$PATH"
+
+rustup="/opt/rust/bin/rustup"
+
+# This is being pinned to a specific date because sometimes nightly will make a change that breaks rustfmt's usage of rustc.
+# We should decide how/when/if we want to advance this date pin.
+toolchain="nightly-2019-02-11"
```

Comment:
I suggest we add this as [a release task](https://github.com/habitat-sh/habitat/blob/master/.github/ISSUE_TEMPLATE/release-checklist.md) to update this. Alternatively, we could make this use the most currently nightly and retry with a few slightly older nightlies dynamically if there's a failure.

Reply:
I like your second suggestion better, as it would ensure that CI is always able run `rustfmt`, vs having a human have to muck around figuring out what the last nightly was that included `rustfmt`.  I'll file an issue to handle this.